internal/repositories/equipment_status.go

```diff
@@ -125,9 +125,15 @@ func (r *equipmentStatusRepository) GetUnavailableEquipmentStatusByEquipmentID(
 
 	return tx.EquipmentStatus.Query().
 		Where(equipmentstatus.HasEquipmentsWith(equipment.IDEQ(equipmentID))).
-		Where(equipmentstatus.HasEquipmentStatusNameWith(equipmentstatusname.IDEQ(4))).
+		Where(
```

Comment:
Could you please add unit tests for the GetUnavailableEquipmentStatusByEquipmentID method? Thanks